<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dimensionamento Residencial Elétrico</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Inserir Dados</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>591</width>
          <height>129</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item alignment="Qt::AlignLeft">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>MS Shell Dlg 2</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string> Cômodo:      </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CBcomodos">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>40</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <item>
              <property name="text">
               <string>Selecionar um Cômodo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sala</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Quarto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cozinha</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Varanda</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Escritório</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Banheiro</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Area de Serviço</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Garagem</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Jardim</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string> Largura: </string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLineEdit" name="lineLargura">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string> Comprimento: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineComprimento"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="ButtonInserir">
             <property name="text">
              <string>Inserir</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ButtonLimpar">
             <property name="text">
              <string>Limpar</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QTableWidget" name="tableDados">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>461</width>
          <height>251</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Cômodo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Área</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Pontos Tomada</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Potência Iluminação</string>
         </property>
        </column>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>140</y>
          <width>121</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string>Ordenação:</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>101</width>
           <height>131</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="Button_OrdenNome">
            <property name="text">
             <string>Nome</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Button_OrdenArea">
            <property name="text">
             <string>Área / Potência</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Button_OrdenQuant">
            <property name="text">
             <string>Quant Tomadas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonOrdenar">
            <property name="text">
             <string>Ordenar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Estatísticas</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>Maior Potência Iluminação</string>
        </property>
        <widget class="QLabel" name="label_maiorPot">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>Menor Potência Iluminação</string>
        </property>
        <widget class="QLabel" name="label_menorPot">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>20</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>Potência Iluminação Média</string>
        </property>
        <widget class="QLabel" name="label_mediaPot">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>Maior Pontos de Tomada</string>
        </property>
        <widget class="QLabel" name="label_maiorQuant">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>130</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>Menor Pontos de Tomada</string>
        </property>
        <widget class="QLabel" name="label_menorQuant">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>130</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>Média de Pontos de Tomada</string>
        </property>
        <widget class="QLabel" name="label_mediaQuant">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>260</y>
          <width>171</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>Potência Ilumanção Total</string>
        </property>
        <widget class="QLabel" name="label_totalPot">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>161</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_9">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>260</y>
          <width>171</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>Pontos de Tomada Total</string>
        </property>
        <widget class="QLabel" name="label_totalQuant">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>161</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_projetista">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>591</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <widget class="QMenu" name="menuSalvar">
     <property name="title">
      <string>Salvar</string>
     </property>
     <property name="icon">
      <iconset resource="arquivoderecursos.qrc">
       <normaloff>:/icones/Imagens/salvar.png</normaloff>:/icones/Imagens/salvar.png</iconset>
     </property>
     <addaction name="action_txt"/>
     <addaction name="action_csv"/>
    </widget>
    <widget class="QMenu" name="menuCarregar">
     <property name="title">
      <string>Carregar</string>
     </property>
     <property name="icon">
      <iconset resource="arquivoderecursos.qrc">
       <normaloff>:/icones/Imagens/carregar.png</normaloff>:/icones/Imagens/carregar.png</iconset>
     </property>
     <addaction name="action_txt_2"/>
     <addaction name="action_csv_2"/>
    </widget>
    <addaction name="menuSalvar"/>
    <addaction name="menuCarregar"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuProjetista">
    <property name="title">
     <string>Projetista</string>
    </property>
    <addaction name="actionEditar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionNormas_e_Recomenda_es"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuProjetista"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSair">
   <property name="icon">
    <iconset resource="arquivoderecursos.qrc">
     <normaloff>:/icones/Imagens/sair.png</normaloff>:/icones/Imagens/sair.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionEditar">
   <property name="icon">
    <iconset resource="arquivoderecursos.qrc">
     <normaloff>:/icones/Imagens/editar.png</normaloff>:/icones/Imagens/editar.png</iconset>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </action>
  <action name="action_txt">
   <property name="icon">
    <iconset resource="arquivoderecursos.qrc">
     <normaloff>:/icones/Imagens/txt.png</normaloff>:/icones/Imagens/txt.png</iconset>
   </property>
   <property name="text">
    <string>.txt</string>
   </property>
  </action>
  <action name="action_csv">
   <property name="icon">
    <iconset resource="arquivoderecursos.qrc">
     <normaloff>:/icones/Imagens/csv.png</normaloff>:/icones/Imagens/csv.png</iconset>
   </property>
   <property name="text">
    <string>.csv</string>
   </property>
  </action>
  <action name="action_txt_2">
   <property name="icon">
    <iconset resource="arquivoderecursos.qrc">
     <normaloff>:/icones/Imagens/txt.png</normaloff>:/icones/Imagens/txt.png</iconset>
   </property>
   <property name="text">
    <string>.txt</string>
   </property>
  </action>
  <action name="action_csv_2">
   <property name="icon">
    <iconset resource="arquivoderecursos.qrc">
     <normaloff>:/icones/Imagens/csv2.png</normaloff>:/icones/Imagens/csv2.png</iconset>
   </property>
   <property name="text">
    <string>.csv</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="icon">
    <iconset resource="arquivoderecursos.qrc">
     <normaloff>:/icones/Imagens/help.png</normaloff>:/icones/Imagens/help.png</iconset>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionNormas_e_Recomenda_es">
   <property name="icon">
    <iconset resource="arquivoderecursos.qrc">
     <normaloff>:/icones/Imagens/normas.png</normaloff>:/icones/Imagens/normas.png</iconset>
   </property>
   <property name="text">
    <string>Normas e Recomendações</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="arquivoderecursos.qrc"/>
 </resources>
 <connections/>
</ui>
